src/m365/spo/commands/navigation/navigation-node-get.ts

```diff
@@ -0,0 +1,81 @@
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils/validation';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { NavigationNode } from './NavigationNode';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  id: string;
+}
+
+class SpoNavigationNodeGetCommand extends SpoCommand {
+  public get name(): string {
+    return commands.NAVIGATION_NODE_GET;
+  }
+
+  public get description(): string {
+    return 'Retrieve information about a specific navigation node';
+  }
+
+  constructor() {
+    super();
+
+    this.#initOptions();
+    this.#initValidators();
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-u, --webUrl <webUrl>'
+      },
+      {
+        option: '--id <id>'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        const id: number = parseInt(args.options.id);
+        if (isNaN(id)) {
+          return `${args.options.id} is not a valid number`;
+        }
+
+        return validation.isValidSharePointUrl(args.options.webUrl);
+      }
+    );
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    if (this.verbose) {
+      logger.logToStderr(`Retrieving information about navigation node with id ${args.options.id}`);
+    }
+
+    const requestOptions: any = {
```

Comment:
I think we should aviod `any` if possible and here we may use `CliRequestOptions` 👍

```suggestion
    const requestOptions: CliRequestOptions = {
```